TanukiKit/Repositories.swift

```diff
@@ -5,60 +5,459 @@ import RequestKit
+//        repository = Repository1(json["repository"] as? [String: AnyObject] ?? [:]) // TODO: Change to repository
+//        project = Repository(json["project"] as? [String: AnyObject] ?? [:]) // TODO: Change to project
+//        totalCommitsCount = json["total_commits_count"] as? Int
+//    }
+//}
+
+//@objc open class Note: NSObject {
+//    open var id: Int?
+//    open var body: String?
+//    open var attachment: String?
+//    open var author: User?
+//    open var createdAt: Date?
+//    open var system: Bool?
+//    open var upvote: Bool?
+//    open var downvote: Bool?
+//    open var noteableID: Int?
+//    open var noteableType: String?
+//
+//    public init(_ json: [String: AnyObject]) {
+//        id = json["id"] as? Int
+//        body = json["body"] as? String
+//        attachment = json["attachment"] as? String
+//        author = User(json["author"] as? [String: AnyObject] ?? [:])
+//        createdAt = Time.rfc3339Date(string: json["created_at"] as? String)
+//        system = json["system"] as? Bool
+//        upvote = json["upvote"] as? Bool
+//        downvote = json["downvote"] as? Bool
+//        noteableID = json["noteable_id"] as? Int
+//        noteableType = json["noteable_type"] as? String
+//    }
+//}
+
 public extension TanukiKit {
     /**
-     Fetches the Repositories for the current user
-     - parameter owner: The user or organization that owns the repositories. If `nil`, fetches repositories for the authenticated user.
-     - parameter page: Current page for repository pagination. `1` by default.
-     - parameter perPage: Number of repositories per page. `100` by default.
+     Fetches the Projects for which the authenticated user is a member.
+     - parameter page: Current page for project pagination. `1` by default.
+     - parameter perPage: Number of projects per page. `100` by default.
+     - parameter archived: Limit by archived status. Default is `""`, set to `true` to enable.
+     - parameter visibility: Limit by visibility `public`, `internal` or `private`. Default is `""`
+     - parameter orderBy: Return projects ordered by `id`, `name`, `path`, `created_at`, `updated_at`, or `last_activity_at` fields. Default is `created_at`.
+     - parameter sort: Return projects sorted in asc or desc order. Default is `desc`.
+     - parameter search: Return list of authorized projects matching the search criteria. Default is `""`
+     - parameter simple: Return only the ID, URL, name, and path of each project. Default is `""`, set to `true` to enable.
      - parameter completion: Callback for the outcome of the fetch.
      */
-    public func repositories(_ session: RequestKitURLSession = URLSession.shared, page: String = "1", perPage: String = "100", completion: @escaping (_ response: Response<[Repository]>) -> Void) -> URLSessionDataTaskProtocol? {
-        let router = RepositoryRouter.readAuthenticatedRepositories(configuration, page, perPage)
+
```

Comment:
Remove empty lines between documentation and method as Xcode does not pick it up otherwise.